core/classes/Core/Output.php

```diff
@@ -53,9 +53,10 @@ public static function getPurified(?string $input, bool $escape_invalid = false)
             $purifierConfig->set('HTML.Doctype', 'XHTML 1.0 Transitional');
             $purifierConfig->set('URI.DisableExternalResources', false);
             $purifierConfig->set('URI.DisableResources', false);
-            $purifierConfig->set('HTML.Allowed', 'u,a,p,b,i,small,blockquote,span[style],span[class],p,strong,em,li,ul,ol,div[align],br,img,figure,figcaption');
-            $purifierConfig->set('CSS.AllowedProperties', ['text-align', 'display', 'float', 'color', 'background-color', 'background', 'font-size', 'font-family', 'text-decoration', 'font-weight', 'font-style', 'font-size']);
+            $purifierConfig->set('HTML.Allowed', 'u,a,p,b,i,small,blockquote,img[style|width|height|alt|src],span[style|class],div[style|align],p,strong,em,li,ul,ol,br,img,figure,figcaption');
+            $purifierConfig->set('CSS.AllowedProperties', ['text-align', 'display', 'float', 'color', 'background-color', 'background', 'font-size', 'font-family', 'text-decoration', 'font-weight', 'font-style', 'font-size', 'width']);
             $purifierConfig->set('CSS.AllowTricky', true);
+            $purifierConfig->set('CSS.MaxImgLength', null);
```

Comment:
Why is this change necessary? According to the HTMLPurifier docs, it opens up to imagecrash attacks (imagine someone adding a height + width of millions of pixels 😄)

Reply:
I've tested it myself with a 50000x50000 image but we can add add a limit.

Reply:
> 

i still believe its a issue, i had way to big image many years ago with made the website extremly slow with being the case @samerton telling i guess